channels.json

```diff
@@ -56,8 +56,8 @@
       "url": "https://blog.tracelog.in/feed"
     },
     {
-      "name": "NamTX",
-      "avatar_url": "",
+      "name": "namtx",
+      "avatar_url": "https://avatars.githubusercontent.com/u/25602820?v=4",
```

Comment:
Hi Nam, sorry but we do not support loading avatars from an URL yet, could you pls add this file to `public/avatars` folder?

Reply:
Yes, Im gonna close this one and open another PR.